### DISCUSSION TITLE
Is GitHub copilot compliant with the National Institute of Standards and Technology (NIST) regulations?

### DISCUSSION THREAD

**Discussion by daniel-utopiacompression** [community/community#129121] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Is GitHub copilot compliant with the National Institute of Standards and Technology (NIST) regulations?
If I use copilot would I be violating the NIST regulations?

**Comment by Akash1134** [comment, marked as answer]
Hi @daniel-utopiacompression ,

We recommend that you consult with your organization's compliance team to ensure adherence to relevant regulations since Copilot's compliance with NIST regulations depends on how it is used within your specific context and the particular NIST standards in question. Please check the [GitHub Trust Center](https://resources.github.com/copilot-trust-center/) to learn more about different aspects of security, privacy, compliance, and transparency iterated on GitHub Copilot.

Thanks!
